README.md

```diff
@@ -1,12 +1,5 @@
 <p align="center"><img width=55% height=55% src="docsimg/jbpm_logo_450px.png"></p>
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-[![GitHub Stars](https://img.shields.io/github/stars/kiegroup/jbpm.svg)](https://github.com/kiegroup/jbpm/stargazers)
-[![GitHub Forks](https://img.shields.io/github/forks/kiegroup/jbpm.svg)](https://github.com/kiegroup/jbpm/network/members)
-[![Pull Requests](https://img.shields.io/github/issues-pr/kiegroup/jbpm.svg?style=flat-square)](https://github.com/kiegroup/jbpm/pulls)
-[![Contributors](https://img.shields.io/github/contributors/kiegroup/jbpm.svg?style=flat-square)](https://github.com/kiegroup/jbpm/graphs/contributors)
-[![License](https://img.shields.io/github/license/kiegroup/jbpm.svg)](https://github.com/kiegroup/jbpm/blob/master/LICENSE-ASL-2.0.txt)
-[![Twitter Follow](https://img.shields.io/twitter/follow/jbossjbpm.svg?label=Follow&style=social)](https://twitter.com/jbossjbpm?lang=en)
```

Comment:
Why are you removing this?

![image](https://user-images.githubusercontent.com/1962786/118481805-2d2bb480-b714-11eb-9f28-05fc2edb3506.png)

Reply:
Because it looks ugly when rendering the file and I do not think it contains valuable info (I might be wrong thought)

Reply:
It was a problem in my browser, restoring it